libraries/AP_Mount/AP_Mount_Backend.cpp

```diff
@@ -934,6 +932,36 @@ bool AP_Mount_Backend::get_angle_target(float& roll_deg, float& pitch_deg, float
     return false;
 }
 
+// return target location if available
+// returns true if a target location is available and fills in target_loc argument
+bool AP_Mount_Backend::get_location_target(Location &_target_loc)
```

Comment:
Please add a `#if AP_SCRIPTING_ENABLED` round this and the same method in the front end. Hopefully that will endup saving a little flash on boards without flash.